docs/installation.rst

```diff
@@ -1,13 +1,101 @@
 Installing Metagov
 ==================
 
-This documentation will walk through how to set up Metagov on an Ubuntu server.
-If you want to set up Metagov for local development, follow the instructions at :doc:`Local Development <../development>` instead.
+This documentation will walk through how to set up Metagov to work with a Driver. If you want to set up Metagov for local development, follow the instructions at :doc:`Local Development <../development>` instead.
 
-Metagov needs to be installed on the same server as your governance "Driver":
+There are three ways to work with Metagov:
 
-* If you're using PolicyKit as your Driver, head over to the `PolicyKit Documentation <https://policykit.readthedocs.io/>`_ for instructions on how to install PolicyKit on your server.
-* If you're using Metagov alongside your existing system, just make sure that you're installing Metagov on the same server. This is necessary because Metagov and your system will communicate over the local network.
+1. Import the Django app directly into a Django-based Driver.
+2. Communicate with a hosted version of Metagov via public (authenticated) HTTP endpoints.
+3. Install your own version of Metagov and communicate locally via HTTP.
```

Comment:
Ah okay, following up from my last comment. I think we should consider dropping explicit support for (3) once we have (2) in place. If people want to run a metagov instance on the same machine as their driver and restrict traffic, fine, but I don't think we need to suggest it or gear the code towards it (e.g. we would be able to remove `/internal` from the routes, and get rid of the instructions for restricting endpoints using an apache2).

Reply:
That seems reasonable to me. I want to leave it in the docs for now since 2 isn't set up yet, though. And once we remove it, I'll probably put it in the wiki or somewhere else with a big warning "deprecated" tag -- would be a shame to delete the instructions entirely.